src/Sound.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include <SFML/Audio.hpp>
+#include <string>
+
+#include "Files.hpp"
+
+class Sound {
+ private:
+  // sound data is not stored directly in sf::Sound, it is stored in sf::SoundBuffer
+  sf::SoundBuffer mBuffer;
```

Comment:
You propably don't need to have member mBuffer, you can create buffer in the constructor and pass it to the mSound member.

Reply:
Please ignore this comment, it's not correct